doc/api/fs.md

```diff
@@ -2578,6 +2578,10 @@ not the file that it refers to.
 <!-- YAML
 added: v0.1.30
 changes:
+  - version: v15.3.0
+    pr-url: https://github.com/nodejs/node/pull/33716
+    description: Accepts an additional `options` object to specify whether
```

Comment:
```suggestion
    description: Accepts a `throwIfNoEntry` option to specify whether
```
because the `options` object already exists at this point.

Ditto with the changes below.

Reply:
Oops, I read that as "an additional `options` property".